src/main/java/org/javarosa/form/api/FormEntryPrompt.java

```diff
@@ -315,6 +316,7 @@ public String getHelpText() {
     /**
      * Helper for getHintText, getHelpText, getConstraintText. Tries to localize text form textID,
      * falls back to innerText if not available.
+     * It may throw XPathTypeMismatchException.
```

Comment:
we should just say `XPathException`  here instead of the more specific `XPathTypeMismatchException` since there are cases when other kinds of XPath errors might result in different XPathExceptions from this method.